Books/client/src/features/Login/index.tsx

```diff
@@ -0,0 +1,15 @@
+import { useState } from 'react'
+import SignIn from './components/SignIn'
+import SignUp from './components/SignUp'
+
+function Login() {
+  // To show sign in page if login is true and show sign up page if login is false
+  const [login, setLogin] = useState(true)
+  return (
+    <div>
+      {login ? <SignIn setLogin={setLogin} /> : <SignUp setLogin={setLogin} />}
```

Comment:
Nitpick formatting issue but do you think you could put each condition on a separate line i.e
```js
{
login
 ? <SignIn setLogin={setLogin} /> 
: <SignUp setLogin={setLogin} />
}
```

Reply:
sure next time i will do that